From 904b999e07ca8c9e5328418913456f9332385167 Mon Sep 17 00:00:00 2001
From: Lars Kellogg-Stedman <lars@oddbit.com>
Date: Fri, 19 Aug 2022 22:32:10 -0400
Subject: [PATCH 7/8] Use correct extension for slapd modules in slapd.conf

The example slapd.conf provided by this package attempts to load
modules with a .la suffix, but the package only provides .so modules.

Update the example slapd.conf to use the correct extension.
---
 servers/slapd/slapd.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/servers/slapd/slapd.conf b/servers/slapd/slapd.conf
index 2fcab71..61f8cd9 100644
--- a/servers/slapd/slapd.conf
+++ b/servers/slapd/slapd.conf
@@ -15,8 +15,8 @@ argsfile	%LOCALSTATEDIR%/run/slapd.args
 
 # Load dynamic backend modules:
 modulepath	%MODULEDIR%
-moduleload	back_mdb.la
-# moduleload	back_ldap.la
+moduleload	back_mdb.so
+# moduleload	back_ldap.so
 
 # Sample security restrictions
 #	Require integrity protection (prevent hijacking)
-- 
2.37.1

